pkgs/applications/office/libreoffice/wrapper.nix

```diff
@@ -1,4 +1,4 @@
-{ libreoffice, runCommand, dbus, bash }:
+{ libreoffice, runCommand, dbus, bash, plugins }:
```

Comment:
you add the argument but don't use it...

Reply:
@symphorien 
I know. This is still a draft. I need to figure out how to complete wrapper.nix. If somebody has a suggestion please let me know.